acl/src/main/java/org/apache/rocketmq/acl/plain/PlainPermissionManager.java

```diff
@@ -266,10 +293,26 @@ public String getAclConfigDataVersion() {
         versionElement.add(accountsMap);
         updateAclConfigMap.put(AclConstants.CONFIG_DATA_VERSION, versionElement);
 
-        dataVersionMap.put(aclFileName, dataVersion);
+        updateDataVersionMap(updateAclConfigMap, dataVersion);
```

Comment:
Why does this place need to get the name of the acl configuration it belongs to according to updateAclConfigMap?

Reply:
It's merging problem, resolved.